handlers/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
 # handlers file for ansible-consul
+- name: reload systemd
+  systemd:
+    name: 'consul'
+    daemon_reload: yes
+
```

Comment:
Thanks for pointing that out.
Issue came up on our side as we run Ansible always in become: true mode.

I also added the when statement attached to all other handlers too.